File: php/arithmetics.php
<?php

$start = microtime(true);

$prev = 1;
for ($i = 1; $i <= 1000; $i++) {
    for ($j = 1; $j <= 10000; $j++) {
        $result = $j * $prev;
        $result = $result + $prev;
        $result = $result - $prev;
        $result = $result / $prev;
        $prev = $j;
    }
}

$end = microtime(true);
printf('%f sec.', $end - $start);

File: python/arithmetics.py
import time

start = time.time()

prev = 1
for i in range(1, 1000):
    for j in range(1, 10000):
        result = j * prev
        result = result + prev
        result = result - prev
        result = result / prev
        prev = j

end = time.time()
print("{0} sec.".format(end - start))
